### PR TITLE
Makes travis compile all the maps in one step so it doesn't get overloaded and backlogged as often.

### PR DESCRIPTION
**Pull request by MrsTonedOne**
We compile all the maps as one step now so that travis only uses 2 jobs, build tools, and maps.

@tgstation/commit-access

### CODE REVIEW

File: tgstation.dme [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 // BEGIN_INCLUDE
 #include "_maps\__MAP_DEFINES.dm"
-#include "_maps\tgstation2.dm"
+#include "_maps\runtimestation.dm"
```
Comment (Jordie0608) [inline tgstation.dme]: Quack!

Comment (MrStonedOne) [inline tgstation.dme]: Feature!

The server tools already force box or meta (depending on servers) unless map rotation happens. So why haven't we made this the default map?

Comment (Mervill) [inline tgstation.dme]: A noob who follows the download/install/launch procedure might not expect to be placed into the debug map. At the very least the tutorial in the readme and on the wiki should be updated with a sentence on switching from runtime to tgstation or another true map.

Comment (MrStonedOne) [inline tgstation.dme]: THIS NEVER HAPPENED

**Comment by Cyberboss** [comment]
Travis emailed you guys after the freeze ended, didn't they. kek.

**Pull request merged**